specification/deviceprovisioningservices/resource-manager/Microsoft.Devices/stable/2022-12-12/iotdps.json

```diff
@@ -0,0 +1,2571 @@
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The resource type."
+        },
+        "properties": {
+          "type": "object",
+          "properties": {
+            "verificationCode": {
+              "type": "string",
+              "description": "Verification code."
+            },
+            "subject": {
+              "type": "string",
+              "description": "Certificate subject."
+            },
+            "expiry": {
+              "type": "string",
+              "description": "Code expiry."
+            },
+            "thumbprint": {
+              "type": "string",
+              "description": "Certificate thumbprint."
+            },
+            "isVerified": {
+              "type": "boolean",
+              "description": "Indicate if the certificate is verified by owner of private key."
+            },
+            "certificate": {
+              "description": "base-64 representation of X509 certificate .cer file or just .pem file content.",
+              "type": "string",
+              "format": "byte"
+            },
+            "created": {
+              "type": "string",
+              "description": "Certificate created time."
+            },
+            "updated": {
+              "type": "string",
+              "description": "Certificate updated time."
+            }
+          }
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "VerificationCodeRequest": {
+      "type": "object",
+      "description": "The JSON-serialized leaf certificate",
+      "properties": {
+        "certificate": {
+          "description": "base-64 representation of X509 certificate .cer file or just .pem file content.",
+          "type": "string"
+        }
+      }
+    }
+  },
+  "parameters": {
+    "subscriptionId": {
+      "name": "subscriptionId",
+      "in": "path",
+      "description": "The subscription identifier.",
+      "required": true,
+      "type": "string"
+    },
+    "api-version": {
+      "name": "api-version",
+      "in": "query",
+      "description": "The version of the API.",
+      "required": true,
+      "type": "string"
+    },
+    "resourceGroupName": {
+      "name": "resourceGroupName",
+      "description": "The name of the resource group that contains the provisioning service.",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "resourceName": {
+      "name": "resourceName",
+      "in": "path",
+      "description": "The name of the provisioning service.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "groupId": {
+      "name": "groupId",
+      "in": "path",
+      "description": "The name of the private link resource",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "privateEndpointConnectionName": {
```

Comment:
>privateEndpointConnectionName [](http://example.com/codeflow?start=5&length=29)

There's a standard definition of this parameter also somewhere, let me find it...

Reply:
[https://github.com/Azure/azure-rest-api-specs/blob/3e27c70e7c02c07b458bc0e94716c3d82d3fdd19/specification/common-types/resource-management/v3/privatelinks.json#LL173C39-L173C39](https://github.com/Azure/azure-rest-api-specs/blob/3e27c70e7c02c07b458bc0e94716c3d82d3fdd19/specification/common-types/resource-management/v3/privatelinks.json#LL173C39-L173C39)

or

[https://github.com/Azure/azure-rest-api-specs/blob/3e27c70e7c02c07b458bc0e94716c3d82d3fdd19/specification/common-types/resource-management/v4/privatelinks.json#LL181C9-L181C9](https://github.com/Azure/azure-rest-api-specs/blob/3e27c70e7c02c07b458bc0e94716c3d82d3fdd19/specification/common-types/resource-management/v4/privatelinks.json#LL181C9-L181C9)